README.md

```diff
@@ -1 +1,5 @@
-# my-first-code-review
```

Comment:
It would be more professional to add a descriptive README file because it makes your project look professional for clients and future employers. I suggest you copy the [template code review](https://github.com/microverseinc/readme-template/blob/master/README.md) from Microverse and paste it into your README file, then customize it to suit your project.